test/test_rb.py

```diff
@@ -2901,6 +2904,58 @@ def test_done_slicesampler(self, strict_length):
             assert (split_trajectories(sample)["next", "done"].sum(-2) == 1).all()
 
 
+@pytest.mark.skipif(not _has_gym, reason="gym required")
+class TestCheckpointers:
+    @pytest.mark.parametrize("storage_type", [LazyMemmapStorage, LazyTensorStorage])
+    @pytest.mark.parametrize(
+        "checkpointer",
+        [FlatStorageCheckpointer, H5StorageCheckpointer, NestedStorageCheckpointer],
+    )
+    def test_simple_env(self, storage_type, checkpointer, tmpdir):
+        env = GymEnv(CARTPOLE_VERSIONED(), device=None)
+        env.set_seed(0)
+        torch.manual_seed(0)
+        collector = SyncDataCollector(
+            env, policy=env.rand_step, total_frames=200, frames_per_batch=22
+        )
+        rb = ReplayBuffer(storage=storage_type(100))
+        rb_test = ReplayBuffer(storage=storage_type(100))
+        rb.storage.checkpointer = checkpointer()
+        rb_test.storage.checkpointer = checkpointer()
+        for i, data in enumerate(collector):
+            rb.extend(data)
+            if i == 0:
```

Comment:
Why do we need to extend text buffer with the data if we are going to `loads` data into buffer just in next line?

Reply:
Removed